pkg/apis/datadoghq/v1alpha1/const.go

```diff
@@ -145,6 +145,10 @@ const (
 	SystemProbeAgentSecurityVolumePath = "/etc/config"
 	SystemProbeSecCompRootVolumeName   = "seccomp-root"
 	SystemProbeSecCompRootVolumePath   = "/host/var/lib/kubelet/seccomp"
+	SystemProbeLibModulesVolumeName    = "modules"
+	SystemProbeLibModulesVolumePath    = "/lib/modules"
```

Comment:
I think we are missing the "kernelHeader folder"
@L3n41c could you confirm?

Reply:
Well, the kernel headers are looked for in `/lib/modules/$(uname -r)/build` which is generally a symlink to `/usr/src/kernel-$(uname-r)`.
So, mounting the `modules` and `src` volumes defined here should be fine.
👍🏻

Reply:
Well, the kernel headers are looked for in `/lib/modules/$(uname -r)/build` which is generally a symlink to `/usr/src/kernel-$(uname-r)`.
So, mounting the `modules` and `src` volumes defined here should be fine.
👍🏻